app/views/shared/_user_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with model: @user, local: true do |f| %>
+  <%= f.label :name if confirmation %>
```

Comment:
I love the work Gus! Can we rename 'confirmation' to 'new_user', or something of the sorts?